app/views/speakers/_form.html.erb

```diff
@@ -11,71 +11,72 @@
     </div>
   <% end %>
 
-  <h2>Speaker</h2>
-
-  <div class="field">
-    <%= form.label :name %>
-    <%= form.text_field :name, id: :speaker_name %>
-  </div>
-
-  <div class="field">
-    <%= form.label :picture %>
-    <%= form.file_field :picture %>
-  </div>
-
-  <div class="field">
-    <%= form.label :email %>
-    <%= form.text_field :email %>
-  </div>
-
-  <div class="field">
-    <%= form.label :company %>
-    <%= form.text_field :company %>
-  </div>
-
-  <div class="field">
-    <%= form.label :biography %>
-    <%= form.text_area :biography %>
-  </div>
+  <div class="form">
+    <h2>Palestrante</h2>
+    <div class="field">
+      <%= form.label :nome %>
+      <%= form.text_field :name, id: :speaker_name, class: :input %>
+    </div>
 
-  <div class="field">
-    <%= form.label :website %>
-    <%= form.text_field :website %>
-  </div>
+    <div class="field">
+      <%= form.label :foto %>
```

Comment:
Here the label is referring to `:picture`, so it should be `<%= form.label :picture %>`, if you want to change the output text of the field you should pass it as a the second arg like `<%= form.label :picture , 'foto' %>`.
You can read more here: http://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/FormHelper.html#method-i-label